dvc/command/get.py

```diff
@@ -10,7 +10,28 @@
 
 
 class CmdGet(CmdBaseNoRepo):
+    def _run_get_url(self):
```

Comment:
I guess good question here is whether or not `--show-url` should also become a part of `repo.get`. Hm... For now, we can leave it as is, since api has get_url anyways. 👍

Reply:
Yes, I wanted to refactor `get_url` and make that part of the internals, but, I left it for now.

Reply:
Though, we could consider splitting it into `_get_url` and `_show_url` here, so that we could test them separately without invoking `main` each time. Kinda like we do in `pipeline show`.

Reply:
I refactored it a bit to make it like `pipeline show`, but, I didn't see any tests where we directly make use of `Cmd*` commands (we do check for isinstance in few places though). `pipeline show` tests also calls `main`.